lib/project_types/script/commands/javy.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require "shopify_cli"
+require_relative "../../../../ext/javy/javy.rb"
+
+module Script
+  class Command
+    class Javy < ShopifyCLI::SubCommand
+      hidden_feature
+
+      prerequisite_task ensure_project_type: :script
+
+      options do |parser, flags|
+        parser.on("--in=IN") { |in_file| flags[:in_file] = in_file }
+        parser.on("--out=OUT") { |out_file| flags[:out_file] = out_file }
```

Comment:
Are we making this required here? Javy defaults to `-o index.wasm`, do we see any value on keeping the same defaults?